### DISCUSSION TITLE
HTTPS/HTTP + CORS ORIGIN VIOLATION

### DISCUSSION THREAD

**Discussion by michael-elkabetz** [FlowiseAI/Flowise#452] [Q&A]
Describe the bug

I have added a chatbot snippet to my website, and I've encountered two issues:

HTTP/HTTPS: My website uses HTTPS, but the chatbot communicates with the server via HTTP.
CORS Origin Violation: The IP of the host that the chatbot communicates with is different from my website's hostname.

How can I fix both of these issues quickly and easily?

**Comment by imcyberuk** [comment]
Anyone find a fix for this?
